[2022-07-05 07:21:40,036] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 07:21:40,065] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 07:21:40,066] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:40,066] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:40,066] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:40,138] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 04:00:00+00:00
[2022-07-05 07:21:40,146] {standard_task_runner.py:52} INFO - Started process 607 to run task
[2022-07-05 07:21:40,159] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T04:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpl3insg7a', '--error-file', '/tmp/tmp_e_gz1eh']
[2022-07-05 07:21:40,160] {standard_task_runner.py:77} INFO - Job 257: Subtask kimuksung_test
[2022-07-05 07:21:40,419] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:40,605] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T04:00:00+00:00
[2022-07-05 07:21:40,606] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:40,607] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:40,637] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T040000, start_date=20220705T072140, end_date=20220705T072140
[2022-07-05 07:21:40,680] {standard_task_runner.py:92} ERROR - Failed to execute job 257 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:40,724] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:40,877] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:51,528] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 07:25:51,571] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 07:25:51,572] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:51,572] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:51,572] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:51,621] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 04:00:00+00:00
[2022-07-05 07:25:51,661] {standard_task_runner.py:52} INFO - Started process 798 to run task
[2022-07-05 07:25:51,679] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T04:00:00+00:00', '--job-id', '288', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp72qrdktu', '--error-file', '/tmp/tmpq1b7vrda']
[2022-07-05 07:25:51,679] {standard_task_runner.py:77} INFO - Job 288: Subtask kimuksung_test
[2022-07-05 07:25:51,887] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:52,000] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T04:00:00+00:00
[2022-07-05 07:25:52,001] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:52,001] {test.py:22} INFO - pubg
[2022-07-05 07:25:52,002] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:52,046] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T040000, start_date=20220705T072551, end_date=20220705T072552
[2022-07-05 07:25:52,097] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:52,148] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:33,620] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 08:06:33,663] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 08:06:33,664] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:33,664] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:33,664] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:33,756] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 04:00:00+00:00
[2022-07-05 08:06:33,773] {standard_task_runner.py:52} INFO - Started process 2198 to run task
[2022-07-05 08:06:33,792] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T04:00:00+00:00', '--job-id', '347', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpxr66b5ro', '--error-file', '/tmp/tmppb_kw75c']
[2022-07-05 08:06:33,793] {standard_task_runner.py:77} INFO - Job 347: Subtask kimuksung_test
[2022-07-05 08:06:34,114] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:34,266] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T04:00:00+00:00
[2022-07-05 08:06:34,269] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:34,270] {test.py:22} INFO - pubg
[2022-07-05 08:06:34,271] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:34,344] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T040000, start_date=20220705T080633, end_date=20220705T080634
[2022-07-05 08:06:34,453] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:34,668] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:46,469] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 08:11:46,498] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [queued]>
[2022-07-05 08:11:46,499] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:46,499] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:46,500] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:46,582] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 04:00:00+00:00
[2022-07-05 08:11:46,611] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T04:00:00+00:00', '--job-id', '411', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmph46u77qt', '--error-file', '/tmp/tmphtvgj9ve']
[2022-07-05 08:11:46,612] {standard_task_runner.py:77} INFO - Job 411: Subtask kimuksung_test
[2022-07-05 08:11:46,588] {standard_task_runner.py:52} INFO - Started process 2498 to run task
[2022-07-05 08:11:47,033] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T04:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:47,406] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T04:00:00+00:00
[2022-07-05 08:11:47,410] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:47,410] {test.py:22} INFO - pubg
[2022-07-05 08:11:47,411] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:47,525] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T040000, start_date=20220705T081146, end_date=20220705T081147
[2022-07-05 08:11:47,655] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:47,790] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
